src/test/java/io/nats/client/utils/TestBase.java

```diff
@@ -218,6 +218,7 @@ public void close() throws Exception {
                 afterJetstream();
             }
             nc.close();
+            ts.close();
```

Comment:
Had re-written some testing stuff 1 PR ago. This shuts down the server when done, otherwise they just stay running after tests are done.